src/components/Book.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeBook } from '../redux/books/Books';
 
-const Book = (props) => {
-  const { book } = props;
+const Book = () => {
+  // const { book } = props;
+  const book = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  const handleRemove = (bookId) => {
+    dispatch(removeBook(bookId));
+    console.log(bookId);
```

Comment:
- [ ] According to the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md):

![image](https://user-images.githubusercontent.com/77069011/223558249-5b5bdf97-79ae-44bb-9921-d5313f6317cf.png)

Please consider removing this `console.log()` statement! 🤓 🔧